Fix uninitialized variable error when doing `npm run unpack`

ian612 opened the pull request:
When running `npm run unpack` in terminal, it throws an error saying `limitToPack` is accessed before initialization. This tweak sets that variable explicitly to `null` to fix the error.

LebombJames commented:
Good catch, I wonder if we can remove the `limitToPack` at the top as well? And it's worth checking if that breaks unpacking via cook too

ian612 commented:
I removed that top `limitToPack` and ran both an `unpack` and a `cook`, and it didn't show any errors on either, so I've made that update (and added some comments)
